input-spec.json

```diff
@@ -1527,7 +1527,10 @@
     {
         "pointer": "/contact/collision_mesh/tessellation_type",
         "type": "string",
-        "options": ["regular", "irregular"],
+        "options": [
```

Comment:
what is this?

Reply:
Those are my changes. Looks like they were formated.